### PR TITLE
Test/node mock reconfigure with available port

### PR DESCRIPTION
**Pull request by sreekanthpuligadda**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-6480 

### Change description ###
Test framework enhancement to resolve port 3001 conflict in pipeline 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by udaydenduluri33 — APPROVED** [review]
@sreekanthpuligadda  thanks for the changes
I can see that you are using portFinder and having port range from 3001 to 3200. 
LGTM

File: test/accessibility/config/pa11yProtractor.conf.js [hunk 1]
```diff
@@ -58,8 +62,9 @@ exports.config = {
 
             });
         } else {
-            MockApp.setServerPort(3001);
+            MockApp.setServerPort(nodeMockPort);
             await MockApp.startServer();
+            console.log("On prepare : mock started on " + nodeMockPort );
```
Comment (rtiru-hmcts) [inline test/accessibility/config/pa11yProtractor.conf.js:67]: Could be the same as in line 52 (But trivial)

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by rtiru-hmcts — APPROVED** [review]
LGTM

**Pull request merged**